about/maintainer_policy/index.markdown

```diff
@@ -50,6 +50,13 @@ We hope it's not too intimidating as we want to add as many qualified Maintainer
  - Have completed at least the minimum requirements for Core Contributors, above
  - Be willing to help review on average 1 pull request a week or more
  - Read the MoveIt pull request guidelines to understand our policies
+ 
+The responsibilities of a MoveIt maintainer include:
+
+ - Review incoming code contributions for style, quality, and overall fit into the goals of MoveIt.
+ - Ensure that Continuous Integration does not break.
+ - Merge pull requests that meet the quality and CI standards above.
+ - Address issues opened up by users.
```

Comment:
I think it would be nice to merge the bullet points of lines 49-52 with the bullet points in L56-59. Let me propose this "squashed" list. 8 bullet points squashed to 6.

- Have proven a good understanding of fundamental parts of the MoveIt code base.
- Have completed at least the minimum requirements for Core Contributors, above.
- Read the MoveIt pull request guidelines to understand our policies.
- Review on average 1 pull request a week or more for style, quality, and overall fit with MoveIt.
- Ensure that Continuous Integration does not break.
- Address open issues.

Reply:
@AndyZe the first section is the criteria for being admitted, the second section is your responsibilities once added. I think we should keep them seperate to clarify what we expect before we add someone, vs after we add someone.